README.md

```diff
@@ -125,4 +135,4 @@ Stanza follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blo
 
 # Other questions?
 
-Check out our [FAQ](/docs/faq.md), or open an issue with your question. We'd love to hear from you.
+Check out our [FAQ](/docs/faq.md), send us an [email](mailto:support.stanza@observiq.com), or open an issue with your question. We'd love to hear from you!
```

Comment:
I know we have a push to get the ticket system up and running. Should we also allow the user to create an issue on this repo? Open source users will almost always prefer to create an issue.

Reply:
This is a question for @jhowellbm